build.gradle

```diff
@@ -1,5 +1,7 @@
 plugins {
-    id "eu.xenit.docker-alfresco" version "4.0.3" apply false
+   id "eu.xenit.docker-alfresco" version "5.0.7" apply false 
```

Comment:
Just adding this plugin is enough. The other plugins are in the same package, so they are automatically also available with the same version.